_posts/2024-12-16-release.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Release 2024.50
+author: Priya
+date: 2024-12-16
+category: Jekyll
+layout: post
+---
+### Product Enhancement :
+**Bulk Imports for SFTP**: Added support for importing multiple files in a single operation, improving efficiency for SFTP data pull. This is an alpha feature and can be enabled by contacting [support@mamoth.io](mailto:support@mamoth.io).
+### Defect Fixes:
+* Resolved an issue causing project load failure due to metadata transformation errors.  
+* Resolved an issue with "Branchout - Keep at the end of pipeline" settings not being retained after modifying the rule.  
```

Comment:
Resolved an issue with "Keep at the end of pipeline" settings not being retained after modifying a Branchout task